app/(afterLogin)/(deal)/deals/manage-deals/page.tsx

```
@@ -0,0 +1,122 @@
+        />
+        <div className="flex gap-2">
+          {/* <Button
+            icon={<LuSettings2 className="text-[#4080f0]" />}
+            className="h-10 border border-[#4080f0] text-[#4080f0]"
+            onClick={() => {
+              setIsFilterOpen(true);
+            }}
+          >
+            Filter
+          </Button> */}
+           <Dropdown
```

ESLint

[error] Delete `·`

(prettier/prettier)